version.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE version SYSTEM "../../../lib/pkp/dtd/pluginVersion.dtd">
+<version>
+	<application>ojs-erc-plugin</application>
+	<type>plugins.generic</type>
+	<release>0.1.0.0</release>
```

Comment:
Are 4 digits required for the version, or can we also stick to `major.minor.bugfix` ?

Reply:
Now we can also stick to 3 digits, the corresponding commit can be found here: https://github.com/o2r-project/ojs-erc-plugin/pull/1/commits/eaea77a44ed699ff3321899fe6290f7d33fe0dd7